TODO/postcss-what-it-is-and-what-it-can-do.md

```diff
@@ -1,72 +1,71 @@
 >* 原文链接 : [PostCSS – What It Is And What It Can Do](https://web-design-weekly.com/2016/06/04/postcss-what-it-is-and-what-it-can-do/)
 * 原文作者 : Jake Bresnehan
 * 译文出自 : [掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者 : 
+* 译者 : [Zheaoli](https://github.com/Zheaoli)
 * 校对者:
 
 
-[PostCSS](http://postcss.org) has been around since September 2013 and has been part of many developers workflow for a while. For those that haven’t had the time to dig into it and put some time aside to understand what it is and what it can do, this post is for you.
+[PostCSS](http://postcss.org) 起源于2013年9月，发展到现在，已经有很多开发者为其贡献代码。 这篇文章是写给那些，还没有来得及花一点时间去了解PostCSS是什么，可以用来干什么的人。
```

Comment:
> For those that haven’t had the time to dig into it and put some time aside to understand what it is and what it can do, this post is for you.

建议考虑中文习惯，避免使用太长的定语，也兼顾原文的语序，可使用“若你还没有……，这篇文章正适合你”的句式